conference.js

```diff
@@ -827,25 +841,14 @@ export default {
 
             this._displayErrorsForCreateInitialLocalTracks(errors);
 
-            return this._setLocalAudioVideoStreams(tracks);
+            return this._setLocalAudioVideoStreams(handleStartAudioMuted(initialOptions, tracks));
```

Comment:
Don't we want to do this with `startWithVideoMuted` case also when gUM resolves after session-accept is sent ?

Reply:
We do it here, I think:
https://github.com/jitsi/jitsi-meet/blob/master/conference.js#L493